src/Illuminate/Foundation/Console/AppNameCommand.php

```diff
@@ -67,6 +67,10 @@ public function __construct(Composer $composer, Filesystem $files)
      */
     public function fire()
     {
+        if (! preg_match('/^[A-Za-z][A-Za-z0-9]*(?:_[A-Za-z0-9]+)*$/', $this->argument('name'))) {
+            return $this->error('Invalid App Name');
```

Comment:
it throws exception now , but i don't know what's the problem for using AppNameCommandException with StyleCI